lms/static/scripts/frontend_apps/test/index-test.js

```diff
@@ -89,6 +89,23 @@ describe('LMS frontend entry', () => {
     });
   });
 
+  it('console logs debug values', () => {
+    const log = sinon.stub(console, 'log');
+
+    try {
+      fakeReadConfig.returns({
+        ...minimalConfig,
+        ...{ debug: { values: { key: 'value' } } },
+      });
+
+      init();
+
+      assert(log.calledWith('key: value'));
```

Comment:
```suggestion
      assert.calledWith(log, 'key: value');
```

You get more useful assert output in the event of failure if using `assert.someCondition(<args>)` instead of `assert(<some boolean condition>)`. The `assert` function on its own is much dumber than how it behaves in Python/pytest.